rust/sdk-processor/src/processors/token_v2_processor.rs

```diff
@@ -0,0 +1,158 @@
+use crate::{
+    config::{
+        db_config::DbConfig,
+        indexer_processor_config::{
+            IndexerProcessorConfig, QUERY_DEFAULT_RETRIES, QUERY_DEFAULT_RETRY_DELAY_MS,
+        },
+        processor_config::{DefaultProcessorConfig, ProcessorConfig},
+    },
+    steps::{
+        common::get_processor_status_saver,
+        token_v2_processor::{
+            token_v2_extractor::TokenV2Extractor, token_v2_storer::TokenV2Storer,
+        },
+    },
+    utils::{
+        chain_id::check_or_update_chain_id,
+        database::{new_db_pool, run_migrations, ArcDbPool},
+        starting_version::get_starting_version,
+    },
+};
+use anyhow::Result;
+use aptos_indexer_processor_sdk::{
+    aptos_indexer_transaction_stream::{TransactionStream, TransactionStreamConfig},
+    builder::ProcessorBuilder,
+    common_steps::{
+        TransactionStreamStep, VersionTrackerStep, DEFAULT_UPDATE_PROCESSOR_STATUS_SECS,
+    },
+    traits::{processor_trait::ProcessorTrait, IntoRunnableStep},
+};
+use serde::{Deserialize, Serialize};
+use tracing::{debug, info};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct TokenV2ProcessorConfig {
+    #[serde(flatten)]
+    pub default_config: DefaultProcessorConfig,
+    #[serde(default = "TokenV2ProcessorConfig::default_query_retries")]
+    pub query_retries: u32,
```

Comment:
actually it seems like a few processors use these two fields (`query_retries` and `query_retry_delay_ms`). Should we just add them as optional to the regular config? 

- Token V2
- Stake
- NFT Metadata

Reply:
Since it's only those 3 and we generally discourage query lookups bc of performance reasons, prefer to keep it in the individual configs. For Token V2, we only do query lookups when parsing old events that don't have complete information. We should look into whether we can remove the lookups completely but haven't had time to.